Makefile

```diff
@@ -119,7 +123,31 @@ OCI_REGISTRY := projects.registry.vmware.com/tce
 
 ##### LINTING TARGETS
 .PHONY: lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint
-check: ensure-deps lint mdlint shellcheck yamllint misspell actionlint urllint imagelint
+check:
+	@make_version=$(shell make --version | grep -E "GNU Make [0-9].[0-9]+" | grep -E -o "[0-9].[0-9]+" | cut -d "." -f1); \
+	if [ "$${make_version}" != "3" ]; then \
+		$(MAKE) check-parallel; \
```

Comment:
Thinking about this some more - we're adding targets so make can call make to call make... can we just move the lines from inside `check-serial` and `check-parallel` to just be called inside this if block instead of calling out to make again?

Reply:
yea, we definitely can do that. not sure why someone might want to be able to call them directly.

Reply:
ok updated